lib/classes/TradeOffer.js

```diff
@@ -797,20 +831,27 @@ TradeOffer.prototype.duplicate = function() {
 	return offer;
 };
 
-TradeOffer.prototype.setMessage = function(message) {
+TradeOffer.prototype.setMessage = function (message) {
 	if (this.id) {
-		throw new Error("Cannot set message in an already-sent offer");
+		throw new Error('Cannot set message in an already-sent offer');
 	}
 
 	this.message = message.toString().substring(0, 128);
 };
 
-TradeOffer.prototype.setToken = function(token) {
+TradeOffer.prototype.setToken = function (token) {
 	if (this.id) {
-		throw new Error("Cannot set token in an already-sent offer");
+		throw new Error('Cannot set token in an already-sent offer');
 	}
 
 	this._token = token;
 };
 
+// A helper to use inside lib/polling.js
+TradeOffer.prototype.needsConfirmation = function () {
+	return this.state === ETradeOfferState.CreatedNeedsConfirmation
+		|| (this.state === ETradeOfferState.Active
+			&& this.confirmationMethod !== EConfirmationMethod.None);
```

Comment:
@DoctorMcKay I just copied it from the `if` statement. What do you think I should add to it?
Also, the logic of this indentation was to make lines shorter. For this case I'm okay with it, not sure about the others. And anyway, if you know the better formatting solution, it would be cool to check it.